tsconfig.json

```diff
@@ -1,15 +1,16 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "ES2022",
     "lib": ["es2015", "es2017", "dom"],
-    "module": "commonjs",
+    "moduleResolution": "Node",
+    "module": "ES2022",
```

Comment:
The storybook monorepo is still using:

```
 "target": "ES2020",
 "module": "CommonJS",
```

This seems like quite a jump.  Is this going to impact our chrome 100+ support target?

Reply:
In the storybook monorepo the tsconfig isn't used to determine the compile target at all

Reply:
Is it being used in this case?  If so, is this the correct target?

Reply:
It might not matter at all, depending on the source code. but let's play it safe and target 2020?

Reply:
will do

Reply:
tomorrow, was a long day 😅

Reply:
done